src/fswAlgorithms/effectorInterfaces/rwMotorTorque/rwMotorTorque.c

```diff
@@ -112,6 +113,12 @@ void Update_rwMotorTorque(rwMotorTorqueConfig *configData, uint64_t callTime, in
     LrInputMsg = CmdTorqueBodyMsg_C_read(&configData->vehControlInMsg);
```

Comment:
:small_red_triangle_down: I know it's not part of the current PR, but I have a question about our approach to there being no linked `vehControlInMsg` message. In this commit you check for linkage of `vehControlIn2Msg` before assigning the value to the torque request, yet in the case of the fist message we didn't. It seems like in the first case we should also check the message is linked before assigning it, not only outputting an error message.

Reply:
The second message is optional, whereas the first must be provided at all times. I think it's weird that the check on that message is in Update rather than in Rebase. Should I move it to Rebase? 

Also, I'm not sure what you mean here: are you saying that I should assign the `LrInputMsg` inside the same if statement in which I check that the message is linked?

Reply:
Agreed, I understand that the second is optional. And sorry, I think I got distracted and posted my comment before adding the rest of my thinking 🤦‍♂️ 

I was thinking along the lines of what additional protections should fsw modules have within them to protect against bad input, resulting in bad output. There's no Basilisk messaging system on the flight system so relying on the messaging system to check for valid data seems to not specifically address the issue. Anyhow, I'll think about this some more. You can leave things as is for the moment.